docs/modules/lists/pages/ordered.adoc

```diff
@@ -43,7 +43,9 @@ If your ordered list includes figures or independent chunks of text, as is often
 This may happen, for example, after figures or multiparagraph steps. 
 In these cases, don't use the `start` attribute to continue the numbering.
 You don't want to have to manually update `start` numbers—or risk having the `start` numbers become wrong—every time you add or remove a step.
-Instead, use a list continuation—a `{plus}` symbol on its own line—between the numbered list items and all other content elements in your list.
+Instead, use a list continuation—a `{plus}` symbol on its own line—between the numbered list items and all other content elements in your list, as shown here:
+
+image::step-numbers-list-continuation.png[Example of continuous step numbering with mixed content elements]
```

Comment:
If the image is saved in ` docs/modules/lists/images` (you'll have to create the `images` folder), then the macro target is correct :+1:

Reply:
Thanks, Sarah. I've rephrased as you suggest. It seems to me, though, that we did need the + between our list items since a blank line before a step (that is, between complex list items like these) seems to reset numbering to 1. But I could be remembering wrong!

Reply:
Thanks @MarciaRieferJohnston for moving and updating the image. It looks perfect now!